opts/options.go

```diff
@@ -89,6 +90,14 @@ var Defaults = func(o *Options) error {
 	return nil
 }
 
+// RoutingTableCleanupInterval is the interval between two runs of the RT cleanup routine.
+func RoutingTableCleanupInterval(i time.Duration) Option {
```

Comment:
nit: maybe call this `RoutingTableCheckInterval` to make it clear that we're not _just_ cleaning. To me, "clean" implies "local cleanup work", not "try to connect to a bunch of peers".

Reply:
Done.